### DISCUSSION TITLE
Can I make a note hyperlink activate a command?

### DISCUSSION THREAD

**Discussion by danalva9603** [freeplane/freeplane#1843] [Help] [answered]
For example, I was thinking of making a floating node that I can click that would hide or unhide hidden nodes.

Is such a thing possible?

**Comment by JJSlote** [comment, marked as answer]
Yes, try:  **_Insert > Link > Add hyperlink to menu item_** 

Rgds - Jerome

**Reply by danalva9603** [reply]
That works. Thank you.
